### PR TITLE
Suggestions for adding to glossary

### PR DESCRIPTION
**Pull request by kallewesterling**
I have created a couple of new glossary sections and drafted some entries based on the repos in our DHRI curriculum as well as requests in the issues opened here. Review, edit, and reject as you like!

(I can't add @javo0101 and @rachelrakov as reviewers as they are not yet on the "people" tab for the DHRI-Curriculum repo but I have invited them and hope I can add them both later.)

### CODE REVIEW

**Comment by smythp** [comment]
> *SQLite* is one implementation of SQL, which requres less work, operates on regular text files and does not require a server connection unlike other implementations such as PostgreSQL and MySQL.

requres > requires, also I wouldn't say it requires less work. Maybe that it's more lightweight. It doesn't use text files (i.e., the DB file is binary), but it's true that unlike other implementations you interface with the DB as a file, not as a server.

Otherwise looks good! Though of course we should have relevant leaders work on these.

**Comment by kallewesterling** [comment]
Thanks for the comment @smythp. Yes, we definitely should. I have basically pulled all the definitions from their workshops so there shouldn't be much too discuss except if they change the workshop content and/or want to add more/different terms!

File: sections/gis.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# GIS
+
+## Glossary  
+
+Please see below a list of key terms and concepts, provided in alphabetical order, from Javier Otero Peña's [Introduction to Mapping using QGIS workshop](https://github.com/DHRI-Curriculum/mapping).
+
+An *attribute* is a characteristic of a feature (see below). Attributes can contain data of different types: strings (text), numerical values, dates, or booleans.
+
+*Feature*: an entity that appears in a layer on a map
```
Comment (javo0101) [inline sections/gis.md]:
```suggestion
*Feature*: an element that appears in a layer and typically has attributes. Each row in the Attribute Table represents a different feature. Each feature can be represented on a map by one (or more) vectors, be it points, lines or polygons.
```

File: sections/gis.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+*Georeferencing*: the process of using geographic data to represent features on a GIS.
+
+*Layers* organizes the data in QGIS. They can be imagined as transparent film sheets with different types of features drawn over them:
```
Comment (javo0101) [inline sections/gis.md]:
```suggestion
*Layers* are "containers" of the data in QGIS. On the map view, layers can be imagined as transparent film sheets that are laid one over another. With the exception of raster layers, each layer contains an Attribute Table, that is, a series of features that in turn have their own attributes. Vector layers can only contain one type of vectors, be it: points, lines or polygons.
```

File: sections/gis.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+An *attribute* is a characteristic of a feature (see below). Attributes can contain data of different types: strings (text), numerical values, dates, or booleans.
+
+*Feature*: an entity that appears in a layer on a map
+
+*Georeferencing*: the process of using geographic data to represent features on a GIS.
+
+*Layers* organizes the data in QGIS. They can be imagined as transparent film sheets with different types of features drawn over them: 
+
+*Raster*: images or maps with a specific resolution that determine how accurate the information is (as well as the file size). They are commonly used for continuous data, such as temperature and elevation.
```
Comment (javo0101) [inline sections/gis.md]:
```suggestion
*Raster*: images of a specific location that represent visually continuous data such as temperature and elevation at a given resolution. Higher resolutions mean more precision but also larger file size. Raster layers have no Attribute Table; the values are stored within the image and represented as different hues or colors. Raster images can also be used for reference or aesthetic purposes (e.g. satellite photos).
```

File: sections/gis.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+*Vector*: a scalable point, line or polygon that can be easily created, edited, or deleted using QGIS. Does not have a specific resolution.
```
Comment (javo0101) [inline sections/gis.md]:
```suggestion
*Vector*: a scalable point, line or polygon that can be easily created, edited, or deleted using GIS. Does not have a specific resolution. In QGIS, vectors are contained in vector layers, that can only contain a single type of vectors. Each vector represents a single feature, although some features may be represented by more than one vector (e.g. the vector of the United States includes the mainland U.S. polygon but also polygons for Alaska and Hawaii).
```

**Pull request merged**